staging/src/k8s.io/sample-apiserver/artifacts/example/deployment.yaml

```diff
@@ -20,8 +20,8 @@ spec:
       - name: wardle-server
         # build from staging/src/k8s.io/sample-apiserver/artifacts/simple-image/Dockerfile
         # or
-        # docker pull gcr.io/kubernetes-e2e-test-images/sample-apiserver:1.17
-        # docker tag gcr.io/kubernetes-e2e-test-images/sample-apiserver:1.17 kube-sample-apiserver:latest
+        # docker pull k8s.gcr.io/e2e-test-images/sample-apiserver:1.17
+        # docker tag k8s.gcr.io/e2e-test-images/sample-apiserver:1.17 kube-sample-apiserver:latest
```

Comment:
could we use version 1.17.4? That version also includes Windows support, so it's generally more available for more people.